docker-compose.yml

```diff
@@ -1,8 +1,7 @@
 frontend:
   image: codeforpoznan/volontulo
   # build: .
-  entrypoint: ng serve --open --host=0.0.0.0
-  # entrypoint: tail -f /dev/null
```

Comment:
why this one line was removed? leave it, please

Reply:
I dont't know when and how does it happen, I'll will fix it.